charts/gpu-metrics-exporter/templates/dcgm-exporter-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
Maybe this configmap should also be created conditionally, if dcgm-exporter will be deployed.